### DISCUSSION TITLE
NMMiner 0.3.03 hashing at 119 KH/s...

### DISCUSSION THREAD

**Discussion by dwightmulcahy** [BitMaker-hub/NerdMiner_v2#502] [General]
So I recently ran across this: https://github.com/NMminer1024/NMMiner

**_"Deeply optimazation for ESP32-S3, max hashrate: 119 KH/s"_**

![image](https://github.com/user-attachments/assets/7a977bde-20d1-4f15-b3dd-665485a8276d)

There isn't any source code (🚩), just the firmware files.  You must have a "license" generated from the device ID that redirects you to PayPal for $2.99.  Searching the interwebs, I get a couple of hits for ads selling these, but not much else.  There is this site that http://nmminer.shop/ is associated with the license generation with not much on it.

I had an extra lilygo-t-display-s3 lying around that wasn't running NerdminerV2 on it, so I downloaded and flashed it with the NMMiner 0.3.03 release and went ahead and popped for the $3 license to see if it worked.

I isolated it from my network and set it up to point to [public-pool.io](https://web.public-pool.io/) and set it on my desk.  It shows that it is mining at 113-119 KH/s on its display (that goes to sleep after 60 seconds--Nice), and public-pool.io stats show that it is doing that, too.

<img width="798" alt="image" src="https://github.com/user-attachments/assets/5609f036-341b-47f2-9941-c7659f67f3f3">

**Comment by dwightmulcahy** [comment]
None that I know of.  This looks to be a side project that was monetized by someone.  I have no idea if this was a fork of this source or what.

**Comment by BitMaker-hub** [comment]
I think this is merely a heltec fork of nerdminer that where able to increase a little bit the hashrate and decided to monetize OpenSource Work. 
This is the worst way to contribute and is not helping on any way to Bitcoin. 
If there is anybody who would like to try to increase NM up to this hash, sounds good to me. hehe

**Comment by Tay21tay** [comment]
How do I pull the money off the miner
